rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/component/ComponentController.java

```diff
@@ -104,6 +104,20 @@
         return new ResponseEntity<>(userHalResource, HttpStatus.OK);
     }
 
+    @PreAuthorize("hasAuthority('WRITE')")
+    @RequestMapping(value = COMPONENTS_URL + "/{id}", method = RequestMethod.PATCH)
+    public ResponseEntity<Resource<Component>> patchComponent(
+            @PathVariable("id") String id,
+            @RequestBody Component updateComponent,
```

Comment:
Do we ignore wrong fields?
In case of wrong body request (component) which contains irregular component data do we just skip them or maybe throw an exception? 
e.g. `400 bad request` or `422 unprocessable entity`

Reply:
At the moment jackson is parsing the body to a `component` object and all non-component properties will be automatically ignored by jackson without throwing an exception.